dotCMS/src/main/java/com/dotcms/security/apps/AppsAPIImpl.java

```diff
@@ -1005,4 +1009,146 @@ public void resetSecrets(final User user)
        appsCache.clearCache();
     }
 
+    /**
+     *
+     * @param key
+     * @param paramAppKeysBySite
+     * @return
+     */
+    public File exportSecrets(final Key key, final boolean exportAll,
+            final Map<String, Set<String>> paramAppKeysBySite, final User user)
+            throws DotDataException, DotSecurityException, IOException {
+
+        if(!userAPI.isCMSAdmin(user)){
```

Comment:
I think you can use user.isAdmin, it is pretty much the same

Reply:
done